stable/redis/README.md

```diff
@@ -167,6 +167,7 @@ The following table lists the configurable parameters of the Redis chart and the
 | `master.readinessProbe.successThreshold`   | Minimum consecutive successes for the probe to be considered successful after having failed (redis master pod) | `1`                                                  |
 | `master.readinessProbe.failureThreshold`   | Minimum consecutive failures for the probe to be considered failed after having succeeded.                     | `5`                                                  |
 | `master.priorityClassName`                 | Redis Master pod priorityClassName                                                                             | {}                                                   |
+| `volumePermissions.enabled`         | Enable init container that changes volume permissions in the registry (for cases where the default k8s `runAsUser` and `fsUser` values do not work)                                                               | `false`                                          |
```

Comment:
Isn't the value being "False" a breaking change?

Reply:
Agree, bumping it

Reply:
Wouldn't it be better to keep backward compatibility and set it to true by default?

Reply:
The issue with this is that `helm install redis` will not work in Openshift out of the box, and that's a User Experience issue that concerns me.

Reply:
Ah, so it does NOT break for regular k8s install by default? If so, okay!

Reply:
Exactly, this was for a special case where the securityContext fsGroup was not enough

Reply:
Anyways, it that would be the case, the `pull-charts-e2e` job would fail.